tf_quant_finance/black_scholes/approximations/BUILD

```diff
@@ -0,0 +1,38 @@
+
+package(default_visibility = ["//tf_quant_finance:__subpackages__"])
+
+py_library(
+    name = "approximations",
+    srcs = ["__init__.py"],
+    srcs_version = "PY3",
+    deps = [
+        ":american_option",
+        # numpy dep,
+        # tensorflow dep,
+    ],
+)
+
+py_library(
+    name = "american_option",
+    srcs = ["american_option.py"],
+    srcs_version = "PY3",
+    deps = [
+        # tensorflow dep,
+        # numpy dep,
```

Comment:
You don't have all the deps:
"//tf_quant_finance/black_scholes:vaniall_prices",
"//tf_quant_finance/black_scholes:implied_vol_newton_root",
"//tf_quant_finance/math"

Reply:
yes! I think this notation of having the specific function was what I was missing!! thanks